index.js

```diff
@@ -128,9 +128,22 @@ exports.createMVPDataFile = function(performanceYear) {
       fs.readFileSync(path.join(__dirname, 'mvp', performanceYear.toString(), 'mvp.json')));
     measuresData = this.getMeasuresData(performanceYear);
   } catch (e) {
-    console.log('QPP measures data not found for year: ' + performanceYear + ' --> ' + e);
+    console.log('QPP mvp / measures data not found for year: ' + performanceYear + ' --> ' + e);
   }
 
+  const mvpIds = [];
+  mvpData.forEach(mvpDataItem => {
+    if (mvpDataItem.mvpId !== 'app1') {
+      mvpIds.push(mvpDataItem.mvpId);
+    }
+  });
+
+  // Reset the allowedPrograms to remove any MVP ID program names (we will add them back on line 173)
+  // This allows for removal of a measure from an MVP data item
+  measuresData.forEach(measure => {
+    measure.allowedPrograms = measure.allowedPrograms ? measure.allowedPrograms.filter(program => !mvpIds.includes(program)) : [];
```

Comment:
This will also have the effect of adding `allowedPrograms` to all measures in measures-data.json that don't already have it, defaulting to an empty array